modules/cnv-preparing-to-install.adoc

```diff
@@ -5,17 +5,11 @@
 [id="cnv-preparing-to-install_{context}"]
 = Preparing to install {CNVProductName}
 
-Before deploying {CNVProductName}:
-
-* Create a namespace called `openshift-cnv`.
-* Create `OperatorGroup` and `CatalogSource` Custom Resource Definition objects
-(CRDs) in the `openshift-cnv` namespace.
+Before deploying {CNVProductName}, create a namespace called `openshift-cnv`.
```

Comment:
IMHO this page is obsolete. The creation of the namespace / project is also mentioned on the `Subscribing to the KubeVirt HyperConverged Cluster Operator catalog` page. The prerequisites and procedure from this page can go there as well. WDYT?

Reply:
It is just a single command at this point, but I'm not sure if we should be switching between the terminal and the UI in the same procedure. I can't have two procedures in one module (both of these files that you referred to are modules, and they are arranged in an assembly that encompasses the entire user story). Thus, I think I might keep this in, at least until it becomes possible to create "openshift-*" namespaces from the UI. I'll consider it further, though. Thanks!